macaw/live_main.py

```diff
@@ -58,10 +73,18 @@ def run(self):
         self.interface.run()
 
 
+parser = argparse.ArgumentParser(description="Run live_main.py file")
```

Comment:
Why do you initialize the parser in the module instead of the `if __name__ == "__main__":` part?

Reply:
I picked this code from other project and didn't realize could move it inside. I've moved it inside now. We don't need global variables.